### ISSUE TITLE
self signed certificate in certificate chain

### ISSUE THREAD

**Issue by thebetterjort** [strapi/strapi#949]
[2018-04-16T14:34:27.048Z] debug GET /users-permissions/routes (16 ms)
[2018-04-16T14:34:27.441Z] error self signed certificate in certificate chain

How do I disable? My corporate firewall inspects SSL traffic. We self sign our key.

**Comment by lauriejim** [comment]
Hi @thebetterjort , can you follow the issue template please. Need more information about node, npm strapi version you use. And also if you are using mongo or sql database.

And please give us more detail on where this issue happen. Actually I can't help you, need more informations.

**Comment by thebetterjort** [comment]
No I will not follow the template.

**Comment by oeddyo** [comment]
@thebetterjort The reason @lauriejim is asking everyone to follow template is to identify problems sooner and avoid back and fourth tedious questions like "what's your npm version". It makes helping people including you and me faster

**Comment by lauriejim** [comment]
Thank you @oeddyo for this explication.

I close this issue cause you don't want give us informations about you issue.

**Comment by drkstr101** [comment]
If it helps at all, I ran into this issue using Digitalocean managed Kubernetes, which requires SSL mode to be on. I was able to connect to db using prisma as a go-between. This is the first time setting up postgres in Digitalocean, so probably it's just a mistake on my part. One thing that is not clear to me is how to provide the path to `ca-certificate.crt` during the initial setup, so this is probably where my mistake is.

Is it just reading whatever is in `.ssh/id_rsa` by default? I do have an ssh key there, but it's not the one I need to use.

PS: Sorry if I missed anything important, I can post a formal issue if I'm unable to get it sorted out. I just wanted to provide some additional context on this issue.
